utils/schema/usersSchema.js

```diff
@@ -4,14 +4,55 @@ const { config } = require('../../config');
 const schema = mongoose.Schema;
 
 const UsersSchema = new schema({
-  FirstName: {
+  documentID: {
+    type: Number,
+    unique: true,
+    trim: true,
+    required: 'Please Supply an identification number',
+  },
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
+  email: {
     type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    required: 'Please Supply an email address',
   },
-  LastName: {
+  contactNumber: {
+    type: Number,
+  },
+  username: {
+    type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    default: function () {
+      const first = this.firstName.split(' ').slice(0, 1).join(' ');
+      const last = this.lastName.split(' ').slice(0, 1).join(' ');
```

Comment:
The code repeats, crate a function for manipulate text fields. Replaces special characters.

Reply:
technical debt: update next milestone